cmd/geth/accountcmd.go

```diff
@@ -291,15 +292,29 @@ func ambiguousAddrRecovery(ks *keystore.KeyStore, err *keystore.AmbiguousAddrErr
 
 // accountCreate creates a new account into the keystore defined by the CLI flags.
 func accountCreate(ctx *cli.Context) error {
-	stack, _ := makeConfigNode(ctx)
+
+	cfg := gethConfig{}
```

Comment:
Please add `Node: defaultNodeConfig()`. That's the one that initializes the default datadir.